src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/Configuration/ApplicationInsightsOutputConfiguration.cs

```diff
@@ -10,5 +10,6 @@ namespace Microsoft.Diagnostics.EventFlow.Configuration
     public class ApplicationInsightsOutputConfiguration: ItemConfiguration
     {
         public string InstrumentationKey { get; set; }
+        public string ConfigurationFilePath { get; set; }
```

Comment:
Yes, but that will be a separate change. Let me just wrap this up and publish on NuGet to get yourself unblocked.